pkg/user/registry/etcd/etcd.go

```diff
@@ -53,7 +52,6 @@ func (r *Etcd) GetUserIdentityMapping(name string) (mapping *api.UserIdentityMap
 func (r *Etcd) CreateOrUpdateUserIdentityMapping(mapping *api.UserIdentityMapping) (*api.UserIdentityMapping, bool, error) {
```

Comment:
This method used to mutate the the passed mapping.  Do callers of this method depend upon that behavior?  

Just want to be sure we check.